packages/sku/entry/csp.js

```diff
@@ -90,13 +108,12 @@ export default function createCSPHandler({
     )};">`;
   };
 
+  /**
+   * @param {string} html
+   * @returns {string}
+   */
   const handleHtml = (html) => {
-    const root = parse(html, {
-      script: true,
-      style: true,
-      pre: true,
-      comment: true,
```

Comment:
Are these all defaults? Should we define and share a single config for all `parse` and `valid` calls?

Reply:
This `parse` call is the only one that cares about more than `script` tags. The others can probably just drop the explicit config actually.

In fact, I'm realising now that [the default actually doesn't preserve HTML comment structure](https://github.com/taoqf/node-html-parser/blob/a3e41a9c9dd8d229a03ace04fff054f9baf7ed88/src/nodes/html.ts#L1001).

Reply:
https://github.com/seek-oss/sku/pull/981/commits/ea6525c34ece493695aa167ccc810c60cdf75d97